lib/grpc/status.ex

```diff
@@ -1,8 +1,28 @@
 defmodule GRPC.Status do
   @moduledoc """
-  Collection of gRPC status.
-
-  Please refer to https://github.com/grpc/grpc/blob/master/include/grpc/impl/codegen/status.h
+  Collection of gRPC status:
+
+   Code |         Status         |                                            Status Message
+  -----:|------------------------|-----------------------------------------------------------------------------------------------------
```

Comment:
```suggestion
  ------|------------------------|-----------------------------------------------------------------------------------------------------
```

Reply:
besides this typo, you need to prepend and append | to all lines  for the table to render correctly

Reply:
You can use `mix docs` to validate this locally

Reply:
> -----:

That is not a typo, It's to align to the right the numbers in that column.

> besides this typo, you need to prepend and append | to all lines for the table to render correctly

Working on that...

Reply:
TIL

Reply:
Fixed [f48e9d6](https://github.com/elixir-grpc/grpc/pull/314/commits/f48e9d6677db4caa481119b67ec5ec4aecd5d917)